### ISSUE TITLE
Direct WSL Support

### ISSUE THREAD

**Issue by MikeGitb** [Microsoft/VSLinux#201]
Sorry if this has been discussed before, but are you planning to provide direct support for WSL? By direct I mean without me needing to separately start a wsl instance, manually starting an ssh server and then sshing into my own machine.

In principle, it should be possible, to just use open-folder on a cmake project and then selecting wsl as a build configuration that automatically starts a bash shell (maybe hidden/integrated into the VS Terminal and execute cmake + make there). I know it is probably not quite that easy to implement, but from a user's perspective, the current solution via ssh seems unnecessary complicated.

**Comment by robotdad** [comment]
Yes we have talked about this before. To date we've treated it as any other remote, now is probably a good time to start taking a look at this again.

**Comment by MikeGitb** [comment]
@Robotdad: Any updates on this?

**Comment by MikeGitb** [comment]
Ping. Any updates? Otherwise I'd just close this issue as there is apparently not a great interest in either the feature itself or in discussing/tracking it here on github.

**Comment by EdorianDark** [comment]
Please do not close it.
I would also be interested in this.

**Comment by MikeGitb** [comment]
@EdorianDark: I don't know if me closing this issue would have any effect on when / if this feature gets worked on. Contrary to other projects like VSCode, there doesn't seem to be enough feedback such that the team could use the issue list to prioritize features.

Of course, there is no harm in leaving this issue open, but if it doesn't serve any practical purpose I like to remove it from my open issue list. 

I see that this issue was now assigned to someone, so let's see if there is any news from MS in the near future.

**Comment by itodirel** [comment]
Hi @MikeGitb, we were planning this feature for Dev15 but we couldn't do it for vcxproj and CMake, only for one, and decided to punt the feature to Dev16. We are planning to do it in Dev16,

**Comment by MikeGitb** [comment]
@itodirel: Thanks, great to hear!
